frontend/src/__tests__/cypress/cypress/e2e/modelRegistry/ModelRegistry.cy.ts

```diff
@@ -1,20 +1,40 @@
 import { mockComponents } from '~/__mocks__/mockComponents';
 import { mockDashboardConfig } from '~/__mocks__/mockDashboardConfig';
+import { mockModelRegistries } from '~/__mocks__/mockModelRegistries';
+import { mockRegisteredModelList } from '~/__mocks__/mockRegisteredModelsList';
+import { mockModelRegistry } from '~/__mocks__/mockModelRegistry';
 import { modelRegistry } from '~/__tests__/cypress/cypress/pages/modelRegistry';
+import { mockModelRegistryRoute } from '~/__mocks__';
 
 type HandlersProps = {
   disableModelRegistryFeature?: boolean;
+  size?: number;
 };
 
-const initIntercepts = ({ disableModelRegistryFeature = false }: HandlersProps) => {
+const initIntercepts = ({ disableModelRegistryFeature = false, size = 0 }: HandlersProps) => {
   cy.interceptOdh(
     'GET /api/config',
     mockDashboardConfig({
       disableModelRegistry: disableModelRegistryFeature,
     }),
   );
   cy.interceptOdh('GET /api/components', { query: { installed: 'true' } }, mockComponents());
+  cy.intercept(
+    '/api/k8s/apis/modelregistry.opendatahub.io/v1alpha1/modelregistries',
+    mockModelRegistries,
+  );
+  cy.intercept(
+    '/api/k8s/apis/modelregistry.opendatahub.io/v1alpha1/namespaces/shared/modelregistries/modelregistry-sample',
+    mockModelRegistry,
+  );
+  cy.intercept(
+    '/api/k8s/apis/route.openshift.io/v1/namespaces/shared/routes/modelregistry-sample',
+    mockModelRegistryRoute,
+  );
+
+  cy.intercept('/api/model_registry/v1alpha2/registered_models', mockRegisteredModelList({ size }));
 };
```

Comment:
Please use `interceptOdh`, `interceptK8s` and `interceptK8sList` vs the plain vanilla `intercept` function.